src/content/fr/2019/compression.md

```diff
@@ -0,0 +1,330 @@
+  <div id="fig10-description" class="visually-hidden">Une capture d'écran d'un rapport Lighthouse montrant une liste de ressources (avec les noms flouté) et montrant la taille et l'économie potentielle. Pour le premier élément, il y a une économie potentiellement importante de 91 KB à 73 KB, tandis que pour d'autres fichiers plus petits de 6 KB ou moins, il y a une économie plus faible de 4 KB à 1 KB.</div>
+  <figcaption id="fig10-caption">Illustration 10. Lighthouse suggestions de compressions.</figcaption>
+</figure>
+
+Comme [HTTP Archive effectue des audits Lighthouse](./methodology#lighthouse) pour chaque page mobile, nous pouvons agréger les scores de tous les sites pour savoir dans quelle mesure il est possible de compresser davantage le contenu. Dans l'ensemble, 62 % des sites web ont réussi cet audit et près de 23 % des sites web ont obtenu une note inférieure à 40. Cela signifie que plus de 1,2 million de sites web pourraient bénéficier d'une compression du texte supplémentaire .
+
+<figure>
+  <a href="/static/images/2019/compression/fig11.png">
+    <img src="/static/images/2019/compression/fig11.png" alt="Illustration 11. Les résultats de l'audit Lighthouse sur la compression de texte" aria-labelledby="fig11-caption" aria-describedby="fig11-description" width="760" height="331" data-width="760" data-height="331" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vQNIyMEGYE_1W0OdFYLIKsxg6M3o_ZsTTuaX73Zzv6Alw4x4D6oH0jdg9BSgw-jy4E-MmX_Qaf-B98W/pubchart?oid=2048155673&amp;format=interactive">
+  </a>
+  <div id="fig11-description" class="visually-hidden">Diagramme à barres empilées montrant que 7,6 % des sites coûtent moins de 10 %, 13,2 % des sites ont un score compris entre 10 et 39 %, 13,7 % des sites ont un score compris entre 40 et 79 %, 2,9 % des sites ont un score compris entre 80 et 99 %, et 62,5 % des sites ont un laissez-passer avec plus de 100 % des textes compressés.</div>
+  <figcaption id="fig11-caption">Illustration 11. Les résultats de l'audit "Activer la compression de texte" de Lighthouse.</figcaption>
+</figure>
+
+Lighthouse indique également combien d'octets pourraient être économisés en permettant la compression du texte. Parmi les sites qui pourraient bénéficier de la compression de texte, 82 % d'entre eux peuvent réduire le poids de leur page de 1 Mo !
+
+<figure>
+  <a href="/static/images/2019/compression/fig12.png">
+    <img src="/static/images/2019/compression/fig12.png" alt="Illustration 12. Audit Lighthouse "activer la compression de texte": économies d'octets potentielles." aria-labelledby="fig12-caption" aria-describedby="fig12-description" width="760" height="331" data-width="760" data-height="331" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vQNIyMEGYE_1W0OdFYLIKsxg6M3o_ZsTTuaX73Zzv6Alw4x4D6oH0jdg9BSgw-jy4E-MmX_Qaf-B98W/pubchart?oid=738657382&amp;format=interactive">
```

Comment:
To to be careful about quotes within quotes as this creates incorrect HTML:

```suggestion
    <img src="/static/images/2019/compression/fig12.png" alt="Illustration 12. Audit Lighthouse 'activer la compression de texte': économies d'octets potentielles." aria-labelledby="fig12-caption" aria-describedby="fig12-description" width="760" height="331" data-width="760" data-height="331" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vQNIyMEGYE_1W0OdFYLIKsxg6M3o_ZsTTuaX73Zzv6Alw4x4D6oH0jdg9BSgw-jy4E-MmX_Qaf-B98W/pubchart?oid=738657382&amp;format=interactive">
```

Also should we translate 'enable text compression' or leave it in English (surrounded by `<i lang='en'></i>` tags) since that is what Lighthouse shows - or is Lighthouse also available in French?

Reply:
Or maybe use chevrons :)